pkg/omf/util/omf.util_bundle_name.fish

```diff
@@ -0,0 +1,3 @@
+function omf.util_bundle_name -a name_or_url
```

Comment:
In this line I prefer `omf.package_name`. Bundle is a different story. Bundle is just a file where packages installed by user are kept for dotfile means.